spa_ui/self_service/client/app/components/dialog-content/dialog-content.html

```diff
@@ -0,0 +1,69 @@
+<div>
+  <div ng-if="!vm.dialog.label">
+    <h2>No Provisioning Dialog Available.</h2>
+  </div>
+  <div ng-if="vm.dialog.label">
+    <div>
+      <h2 class="text-capitalize no-wrap">{{ ::vm.dialog.label }} </h2>
+      <h4>{{ ::vm.dialog.description }}</h4>
+    </div>
+    <tabset>
+      <tab ng-repeat="dialogTab in vm.dialog.dialog_tabs" heading="{{ dialogTab.label }}">
```

Comment:
on the original marketplace details state there were provisions that
indicated multiple provision dialog blocks could be present.  it is for
this reason I left it bound, if this is not the case, no issues with one
off binding

On Wed, Oct 14, 2015 at 10:47 AM, Michael Stack <notifications@github.com>
wrote:

> In
> spa_ui/self_service/client/app/components/dialog-content/dialog-content.html
> <https://github.com/ManageIQ/manageiq/pull/4845#discussion_r42002864>:
>
> > @@ -0,0 +1,69 @@
> > +<div>
> > +  <div ng-if="!vm.dialog.label">
> > +    <h2>No Provisioning Dialog Available.</h2>
> > +  </div>
> > +  <div ng-if="vm.dialog.label">
> > +    <div>
> > +      <h2 class="text-capitalize no-wrap">{{ ::vm.dialog.label }} </h2>
> > +      <h4>{{ ::vm.dialog.description }}</h4>
> > +    </div>
> > +    <tabset>
> > +      <tab ng-repeat="dialogTab in vm.dialog.dialog_tabs" heading="{{ dialogTab.label }}">
>
> Do the label change as the dialog is filled out? Can the dialogTab.label
> be once bound to be removed from the $watch list?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ManageIQ/manageiq/pull/4845/files#r42002864>.
>